packages/js/product-editor/src/components/button-with-dropdown-menu/index.tsx

```diff
@@ -28,10 +28,13 @@ export const ButtonWithDropdownMenu: React.FC<
 		position: 'bottom left left',
 		offset: 0,
 	},
+	className,
 } ) => {
 	return (
 		<Flex
-			className="woocommerce-button-with-dropdown-menu"
+			className={ `woocommerce-button-with-dropdown-menu${
+				className?.length ? ' ' + className : ''
+			}` }
```

Comment:
Can we use `classnames` instead of the ternary operator?
The code would look like this:
```
className={ classnames(
    'woocommerce-button-with-dropdown-menu',
	className
) }
```

Reply:
Thanks Fer! I just merged the pull request, but we can update it in a follow-up. Using the classname lib becomes simpler and more readable.